plugins/auth-backend/src/providers/cloudflare-access/provider.ts

```diff
@@ -259,8 +258,20 @@ export class CloudflareAccessAuthProvider implements AuthProviderRouteHandlers {
     const verifyResult = await jwtVerify(jwt, this.jwtKeySet, {
       issuer: `https://${this.teamName}.cloudflareaccess.com`,
     });
-    const sub = verifyResult.payload.sub;
-    const cfAccessResultStr = await this.cache?.get(`${CACHE_PREFIX}/${sub}`);
+
+    const isServiceToken = verifyResult.payload.sub === '';
```

Comment:
is it _exactly_ the empty string in real production scenarios, or do you want to look for any falsy value here (such as the `sub` not existing at all)?

Reply:
Will update.